### ISSUE TITLE
Blocking - An observation

### ISSUE THREAD

**Issue by eaba** [mgp25/Chat-API#1946]
Make sure these boxes are checked before submitting your issue -- thank you!

- [ x] You are using PHP > 5.6
- [ x] You have installed all extensions and modules listed in [here](https://github.com/mgp25/Chat-API/wiki/Dependencies). PHP Protobuf and Curve25519 are required!
- [x ] You have checked if someone has already asked the same issue you have.
- [ x] You have read the [wiki](https://github.com/mgp25/Chat-API/wiki).
- [ x] Is not a programming question.
- [ x] You are using latest Chat API code.

### Error
Post your error below:
Hello @mgp25 @0xTryCatch this is more of an observation than an issue and I need your comment as regard this:
Now, there have been issues by users who claim that their numbers got blocked without ever sending a message (Me too, i only synced and got blocked), could this be due to end-to-end encryption?

Because i have observed that once my number encryption status is updated to "Messages you send to this chat are secured with end-to-end encryption" without sending any message the number the api number got blocked!

So i am guessing we are missing something in the process, please kindly put me through!

Thanks!

### Debug log
Post your debug log below:

```xml
YOUR DEBUG LOG HERE
```

**Comment by eaba** [comment]
@0xTryCatch at least say something?

**Comment by mgp25** [comment]
wrong

**Comment by eaba** [comment]
Thanks!
@mgp25 i sent you an email going to two weeks now and you have said nothing! Cant you just oblige me, kindly!

**Comment by vnay92** [comment]
So, I have this number I'm using to read any important mails and send it as a WhatsApp Message to my number. This is a daemon that has been running on a Raspberry Pi and Ubuntu Mate.

And guess what? It works perfectly! There was this issue of getting double messages. But that was fixed with a small one line change. [This solved it](https://github.com/mgp25/Chat-API/issues/1791).

I had registered this number using registerTool.php , got my passwords, and it works. Works very well. But, the problem isn't with the Chat API. I think that the problem is with registerTool.php. WhatsApp could  figure out if a numbers is registered using registerTool.php (new numbers) and may block them.

I'm sure @mgp25, @0xTryCatch and others have a fix for this. But, let's be patient and wait for the next release! No point in creating tonnes of issues for the same thing. It will definitely irritate them.

Basically, what I'm saying is, Chat API works like magic! But, registerTool is where the problem is. If by chance, you have any old registered numbers, "keep them safe", new numbers will get blocked. And, wait for the next release!

Peace!

**Comment by q4z1** [comment]
Same here: git cloned the repo yesterday, registered a number with the exampleRegister.php (nothing else done) - waited 24 hours - number got blocked! I really have doubts, that its the "user" who does something wrong.

**Comment by eaba** [comment]
@vnay92 I will agree with you to an extent, yes the registration could be where the problem is!
I have a new number that is still working, I have been care-free with this number and yet WhatsApp refuses to ban it! I deleted one or more parameters while registering!
And thats what led to my observation!

**Comment by mgp25** [comment]
Im done with this
